content/en/docs/user-guides/how-to/directory-structure.md

```diff
@@ -51,7 +56,9 @@ The overall directory structure for `notation` is summarized as follows.
 ```
 {NOTATION_BIN}
 └── notation
-
+{NOTATION_CACHE}
+└── crl
+    └── {sha256-hash-of-the-CRL-file}
```

Comment:
```suggestion
    └── {sha256-hash-of-the-CRL-Downloading-URL}
```

Actually, it is the hash of the downloading URL because before we download the CRL, the only thing we know is the URL. Then, we can hash the URL to try to find the cache.